spec/models/item_spec.rb

```diff
@@ -8,32 +8,58 @@
   end
 
   describe 'validations' do
-    it { should validate_presence_of(:name)}
-    it { should validate_presence_of(:description)}
-    it { should validate_presence_of(:unit_price)}
-    it { should validate_presence_of(:merchant_id)}
+    it { should validate_presence_of(:name) }
+    it { should validate_presence_of(:description) }
```

Comment:
Looks like we doubled down on this.... but our code was pretty much identical. :D